e2e/utils.go

```diff
@@ -1732,3 +1732,13 @@ func rwopMayFail(err error) bool {
 
 	return !rwopSupported
 }
+
+// getConfigFile returns the passed config file location if it exists, else
+// returns the old location of the config file under 'examples/' directory.
+func getConfigFile(configFile string) string {
```

Comment:
This works, but it could be done cleaner. The function should not be specific for `cephConfconfigMap`, I think.

Could you make it like this:
```go
func getConfigFile(filename, preferred, fallback string) string {
```

If the file does not exist in the `preferred` directory, check the `fallback` directory instead. Maybe even `panic()` if the file can not be found at all?

Reply:
Yeah this is much cleaner. Will update it.

I think `panic()` is not required in this section as we get the appropriate error in the `Do(action)` part based on the file's existence and `allowMissing` value.